### PR TITLE
FOSSC-88 alsa-utils

### PR DESCRIPTION
**Pull request by dierberg**

### CODE REVIEW

File: analysed-packages/alsa-utils/version-1.2.6/alsa-utils-1.2.6-SPDX2TV.spdx [hunk 1]
```diff
@@ -0,0 +1,5693 @@
...
+Copyright 1992-2018 Free Software Foundation, Inc. </text>
+
+##File
+
+FileName: alsa-utils-1.2.6.tar.bz2/alsa-utils-1.2.6.tar/alsa-utils-1.2.6/alsaucm/usecase.c
+SPDXID: SPDXRef-item441945
+FileChecksum: SHA1: 058b0cd5d4ffbd20db070f611efa27c3f2fa5ab9
+FileChecksum: SHA256: 949d5690c26dd5a39210eb41fe8cf5c7eca362ec353a03802b6c49a955187a1c
+FileChecksum: MD5: 6e60b1ca0197ee76b7d0ba8079900378
+LicenseConcluded: LicenseRef-LGPL-2.0-or-later-c4a20a7c9332c01337a88843c850c213
```
Comment (OliverFendt) [inline analysed-packages/alsa-utils/version-1.2.6/alsa-utils-1.2.6-SPDX2TV.spdx:1795]: E.g. here you have only the boilerplate of LGPL-2.0-or-later as license text conlude, please correct this. There might ba also other files with the wrong license text

File: analysed-packages/alsa-utils/version-1.2.6/alsa-utils-1.2.6-OSS-disclosure.txt [hunk 2]
```diff
@@ -0,0 +1,1296 @@
...
+
+This file, Rules-quot, and its auxiliary files (listed under
+DISTFILES.common.extra1) are free software; the Free Software Foundation
+gives unlimited permission to use, copy, distribute, and modify them.
+
+--------------------------------------------------------------------------------
+
+FSFUL
+
+This configure script is free software; the Free Software Foundation gives
```
Comment (OliverFendt) [inline analysed-packages/alsa-utils/version-1.2.6/alsa-utils-1.2.6-OSS-disclosure.txt:770]: please add additionally:
This config.status script is free software; the Free Software Foundation
gives unlimited permission to copy, distribute and modify it.
See file "configure"

File: analysed-packages/alsa-utils/version-1.2.6/alsa-utils-1.2.6-SPDX2TV.spdx
Comment (OliverFendt) [inline analysed-packages/alsa-utils/version-1.2.6/alsa-utils-1.2.6-SPDX2TV.spdx:1]: Please resolve in this file my comments in the disclosure file too

File: analysed-packages/alsa-utils/version-1.2.6/alsa-utils-1.2.6.spdx.json
Comment (OliverFendt) [inline analysed-packages/alsa-utils/version-1.2.6/alsa-utils-1.2.6.spdx.json:1]: Please resolve in this file my comments in the disclosure file and tag value file, too

File: analysed-packages/alsa-utils/version-1.2.6/alsa-utils-1.2.6.spdx.rdf.xml
Comment (OliverFendt) [inline analysed-packages/alsa-utils/version-1.2.6/alsa-utils-1.2.6.spdx.rdf.xml:1]: Please resolve in this file my comments in the disclosure file and tag value file, too

File: analysed-packages/alsa-utils/version-1.2.6/alsa-utils-1.2.6.spdx.yaml
Comment (OliverFendt) [inline analysed-packages/alsa-utils/version-1.2.6/alsa-utils-1.2.6.spdx.yaml:1]: Please resolve in this file my comments in the disclosure file and tag value file, too

**Review by OliverFendt — CHANGES_REQUESTED** [review]
@dierberg  Awesome great work: I only have a few comments

File: analysed-packages/alsa-utils/version-1.2.6/alsa-utils-1.2.6-OSS-disclosure.txt [hunk 3]
```diff
@@ -0,0 +1,1296 @@
...
+Copyright (C) 2005 Free Software Foundation, Inc.
+Copyright (C) 2005 THE alsa-utils'S COPYRIGHT HOLDER
+Copyright (C) 2011 The ALSA Team
+Copyright (C) 2015 Caleb Crome
+Copyright (C) 2013-2015 Intel Corporation
+Copyright (c) 2010 by Jaroslav Kysela <perex@perex.cz>
+Copyright (c) 1999-2000 by Jaroslav Kysela <perex@perex.cz>
+Copyright (C) 2007 Free Software Foundation.
+Copyright (c) by Jaroslav Kysela <perex@perex.cz>
+Copyright (c) by Jaroslav Kysela <perex@perex.cz> Based on vplay program by
```
[the remark this answers is not in the document]
Comment (dierberg) [reply analysed-packages/alsa-utils/version-1.2.6/alsa-utils-1.2.6-OSS-disclosure.txt:2294]: Full copyright is: " ... Based on vplay programm by Michael Beck" I would like to have that reference (M.Beck is a former colleague of mine)

**Review by OliverFendt — APPROVED** [review]
Great work, thank you vrey much @dierberg

**Pull request merged**